### ISSUE TITLE
Finetuning Configuration

### ISSUE THREAD

**Issue by aaronabebe** [hugofloresgarcia/vampnet#4]
Hi Hugo! 
First off thanks a lot for the great work, open source rules! 

I have a question regarding finetuning, I currently trained a coarse model on a single album. After 300 epochs I noticed that the train loss was still going down. Are the defaults you provide in the generated `.yaml`'s in `/conf` the ideal configurations? 

From your own finetuning, did you also use 500 epochs per model or less/more? And does the c2f model need the same amount of training or more/less (in general)? 
I also checked the paper but you never mention finetuning configurations. 

All the best, 
Aaron

**Comment by hugofloresgarcia** [comment]
Hi Aaron! 

I haven't experimented much with the max epochs for finetuning a model yet, so definitely feel free to finetune for longer than 500 epochs and let me know how it works out!

For the fine tuning demo, I trained the coarse model for max 500 epochs and the c2f model for max 300 epochs (100 steps per epoch if I'm not mistaken). 

My intuition is that the longer you train it for, the more that generated samples will "cling" to the finetuning dataset. And how 'overfit' it is to specific songs in the dataset will vary depending on the number of steps you do while finetuning as well as the size of the finetuning dataset itself.


Thanks for trying out the model, and I'd love to hear samples from the model you finetuned!
